tests/test_logging.py

```diff
@@ -280,6 +301,9 @@ def some_route():
                 "url": "http://localhost/",
                 "method": "GET",
                 "endpoint": "some_route",
+                "host": "localhost",
+                "path": "/",
+                "user_agent": "Werkzeug/3.0.1",
```

Comment:
This will "break" every time we update bump werkzeug won't it? Suggest either using `mock.ANY` or even e.g. `AnyStringMatching("Werkzeug.*")`

Reply:
Good point. Updated with your suggestion.